packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -156,10 +165,20 @@ const removeForumEventVoteQuery = gql`
   }
 `;
 
-const maxVotePos = SLIDER_WIDTH - USER_IMAGE_SIZE
+const maxVotePos = (SLIDER_MAX_WIDTH - USER_IMAGE_SIZE) / SLIDER_MAX_WIDTH
```

Comment:
I attempted to convert everything to percentages, let me know if I missed anything!